cumulus_library_data_metrics/q_valid_us_core_v4/README.md

```diff
@@ -55,6 +55,13 @@ All possible rows for the resource in question.
 In the case of Observation profiles,
 it's just the count of rows with the category in question.
 
+### A note on the DiagnosticReport profile
+
+The DiagnosticReport profiles mark `presentedForm` as "Must Support".
+
+But since Cumulus ETL strips these fields,
+this metric does not examine them.
```

Comment:
would be be worth explaining why we strip them?

Reply:
Added a mention of PHI to this and other similar sentences in this file